blog-service/2023-05-26-metrics.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Display Summary Statistics in Legend (Metrics)
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - sumo logic
+  - service release notes
+  - open source
```

Comment:
These three were keywords were written for our initial release note in October. The other two are perfect cause they're very specific to this topic 👍

```suggestion
  - sumo logic
  - service release notes
  - open source
```

Reply:
Thanks Kim.  We don't need them anymore.